### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by adrianmoisey**

### CODE REVIEW

**Review by zacblazic — CHANGES_REQUESTED** [review]
Do this.

File: README.md [hunk 1]
```diff
@@ -2,3 +2,6 @@
 
 in blah
 
+
```
Comment (zacblazic) [inline README.md:5]: Add more detail please.

**Review by zacblazic — CHANGES_REQUESTED** [review]

**Comment by adrianmoisey** [comment]
Hullo

**Comment by adrianmoisey** [comment]
hullo

**Comment by adrianmoisey** [comment]
hul"lo

**Pull request merged**